### PR TITLE
Feature/various fixes for deployment

### PR DESCRIPTION
**Pull request by jhedev**

### CODE REVIEW

File: cmd/snapshotter/main.go [hunk 1]
```diff
@@ -25,7 +26,17 @@ func main() {
 	)
 	flag.Parse()
 
-	sess := session.New()
+	creds := credentials.NewEnvCredentials()
```
Comment (philippfranke) [inline cmd/snapshotter/main.go]: I think we should not mix `env` and `flags`. Let's add either flags for the AWS credentials or env variables.

**Review by philippfranke — CHANGES_REQUESTED** [review]
What do you think about my comment?

File: cmd/snapshotter/main.go (hunk 1 above)
Comment (jhedev) [reply cmd/snapshotter/main.go]: Don't have a strong opinion about that. So if you think we should not mix it I'm fine with updating this.

Comment (philippfranke) [reply cmd/snapshotter/main.go]: Then I'd prefer flags 👍

**Comment by jhedev** [comment]
@philippfranke fixed and rebased

**Review by philippfranke — APPROVED** [review]
👍

**Pull request merged**